src/ReportCommand.php

```diff
@@ -141,6 +174,25 @@ protected function outputFormattedLicenses(OutputInterface $output, array $licen
                 ]);
             }
             $licenseTable->render();
+
+            if ($input->getOption('show-packages') || $output->isVerbose()) {
+                $output->writeln('');
+
+                if ($input->getOption('grouped')) {
+                    $output->write('packages: ');
+                    $packages = [];
+                    foreach ($dependencies as $dependency) {
+                        $packages[] = $dependency->getName();
+                    }
+
+                    $output->write(implode(', ', $packages));
+                    continue;
+                }
+
+                foreach ($dependencies as $dependency) {
+                    $output->writeln(sprintf('%s (%s)', $dependency->getName(), $dependency->getVersion()));
+                }
+            }
```

Comment:
This snipped lends itself to being extracted into a helper method, that just returns the string to print. That also helps to keep the `outputFormattedLicenses` method shorter as it is quite bloated already.